DigitalLearningSolutions.Web/Styles/home/homeShared.scss

```diff
@@ -0,0 +1,34 @@
+@import "~nhsuk-frontend/packages/core/all";
+
+.landing-page__body-text {
+  @extend .nhsuk-body-m;

```

Comment:
If this is just an extension, is there a reason we're not using the nhsuk-body-m class directly to avoid an unnecessary layer of nesting? [same question for content-area__heading]

Reply:
Consistency was my reasoning - I guess we want the body text to be the same across the landing page, so if we should decide to change the size we only need to make one change rather than finding all the nhsuk-body-m sized text across the 3 views.

Reply:
Hmm.. I feel like the pages are quite different in terms of layout, so I don't agree with the "consistency" aspect here. I can see someone wanting to modify the Welcome page body text and accidentally affecting the other pages too. However, I can see how the consistency matters within the Learning Content cards for example or the common elements that you've put into a partial. ⭐